cekit/descriptor/packages.py

```diff
@@ -69,28 +71,34 @@ def __init__(self, descriptor):
             descriptor['filename'] = '%s.repo' % descriptor['name'].replace(' ', '_')
 
         if not (('url' in descriptor) ^
-                ('odcs' in descriptor) ^
                 ('id' in descriptor) ^
-                ('rpm' in descriptor)):
+                ('rpm' in descriptor) ^
+                ('content_sets' in descriptor) ^
+                ('content_sets_file' in descriptor)):
             raise CekitError("Repository '%s' is invalid, you can use only one of "
-                             "['id', 'odcs', 'rpm', 'url']"
+                             "['id', 'content_sets_file', 'content_sets', 'rpm', 'url']"
```

Comment:
I'm not sure if we want to remove odcs, this is a general type. content_sets is very specific to our environment.

Reply:
We dont have any use case for it now. And it was never used at the end, so I will propose to not carry it over.